app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -58,8 +58,8 @@ public function update(User $user): bool
     /**
      * Determine whether the authenticated user can lock other user accounts.
      *
-     * @param  User $user  The resource entity from the authenticated user.
-     * @param  User $model The resource entity from the given user.
+     * @param User $user The resource entity from the authenticated user.
+     * @param User $model The resource entity from the given user.
```

Comment:
Hey,

Can u align the User class with the same start as the return type in the docblock.
Also the start of the description text.